packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -10,6 +10,7 @@ const {
   sanitize,
 } = require('@strapi/utils');
 const { ValidationError } = require('@strapi/utils').errors;
+const { transformParamsToQuery } = require('@strapi/utils/lib/convert-query-params');
```

Comment:
Those requires should be avoided when possible. use require('@strapi/utils').....

Reply:
Yes, I was wondering as I copy paste and usually don't do that 👌